modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -16,14 +21,21 @@ ifdef::c3[]
 You must create the `install-config.yaml` and the `agent-config.yaml` configuration files so that you can use the Agent-based Installer to generate a bootable ISO image. The Agent-based installation comprises a bootable ISO that has the Assisted discovery agent and the Assisted Service. Both of these components are required to perform the cluster installation, but the latter component runs on only one of the hosts.
 endif::c3[]
 
-ifndef::c3[]
+ifdef::pca[]
+[id="creating-config-files-cluster-install-pca_{context}"]
+= Creating configuration files for installing a cluster on {oci-pca-short}
+
+You must create the `install-config.yaml` and the `agent-config.yaml` configuration files so that you can use the Agent-based Installer to generate a bootable ISO image. The Agent-based installation comprises a bootable ISO that has the Assisted discovery agent and the Assisted Service. Both of these components are required to perform the cluster installation, but the latter component runs on only one of the hosts.
+endif::pca[]
+
+ifndef::pca,c3[]
 [id="creating-config-files-cluster-install-oci_{context}"]
 = Creating configuration files for installing a cluster on {oci}
 
 You must create the `install-config.yaml` and the `agent-config.yaml` configuration files so that you can use the Agent-based Installer to generate a bootable ISO image. The Agent-based installation comprises a bootable ISO that has the Assisted discovery agent and the Assisted Service. Both of these components are required to perform the cluster installation, but the latter component runs on only one of the hosts.
 
-At a later stage, you must follow the steps in the Oracle documentation for uploading your generated Agent ISO image to a default Oracle Object Storage bucket, which is the initial step for integrating your {product-title} cluster on {oci-first}.
-endif::c3[]
+At a later stage, you must follow the steps in the Oracle documentation for uploading your generated agent ISO image to a default Oracle Object Storage bucket, which is the initial step for integrating your {product-title} cluster on {oci-first}.
```

Comment:
> At a later stage, you must follow the steps in the Oracle documentation 
When do we tell folks that they need to do this? This sentence seems a bit vague to me.